server/src/Schemas/messages.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+const ObjectId = Schema.ObjectId;
+
+const MessageSchema = new Schema({
+  message: String,
+  read: Boolean,
+  timestamp: Date,
```

Comment:
Using the date as a timestamp.
Are you talking about this?
@nimish-gupta

Reply:
so there is a default timestamp option in mongoose, you can use that.